app/sections/image-with-text.tsx

```diff
@@ -0,0 +1,191 @@
+          </div>
+          <div className='w-3/5 flex h-full py-14 justify-end sm-max:order-first sm-max:w-full sm-max:py-10 sm-max:pb-0 sm-max:justify-center'>
+            <div className='z-10 sm:translate-x-8 sm:translate-y-8 sm-max:z-10 sm-max:translate-x-9 sm-max:translate-y-9'>
+              <Image data={imageFisrt} loading={loading} className='sm:object-contain sm-max:object-contain'/>
+            </div>
+            <div>
+              <Image data={imageSecond} loading={loading} className='sm:object-contain sm-max:object-contain'/>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+});
+
+export default ImageWithText;
+
+export let schema: HydrogenComponentSchema = {
+  type: 'image-with-text',
+  title: 'Image with text',
+  toolbar: ['general-settings', ['duplicate', 'delete']],
+  inspector: [
+    {
+      group: 'Image',
+      inputs: [
+        {
+          type: 'image',
+          name: 'imageFisrt',
+          label: 'Image 1',
+          defaultValue:{
+            url: 'https://thang-weaverse-test-shop.myshopify.com/cdn/shop/files/gift-card-envelopes.jpg?v=1696497046&width=1100',
+            altText: 'imageIndex',
+          }
+        }, {
+          type: 'image',
+          name: 'imageSecond',
+          label: 'Image 2',
+          defaultValue:{
+            url: 'https://thang-weaverse-test-shop.myshopify.com/cdn/shop/files/gift-card.jpg?v=1696497401&width=1100',
+            altText: 'imageIndex',
+          }
+        },
+        {
+          type: 'text',
+          name: 'subHeading',
+          label: 'Subheading',
+          defaultValue: 'Subheading',
+          placeholder: 'Subheading',
+        },
+        {
+          type: 'toggle-group',
+          label: 'Subheading size',
+          name: 'subHeadingSize',
+          configs: {
+            options: [
+              { label: 'XS', value: '14px' },
+              { label: 'S', value: '16px' },
+              { label: 'M', value: '18px' },
+              { label: 'L', value: '20px' },
+              { label: 'XL', value: '22px' },
+            ],
+          },
+          defaultValue: '16px',
+        },
+        {
+          type: 'text',
+          name: 'heading',
+          label: 'Heading',
+          defaultValue: 'Heading for Image',
+          placeholder: 'Heading for image section',
+        },
+        {
+          type: 'toggle-group',
+          label: 'Heading size',
+          name: 'headingSize',
+          configs: {
+            options: [
+              { label: 'XS', value: '22px' },
+              { label: 'S', value: '24px' },
+              { label: 'M', value: '26px' },
+              { label: 'L', value: '28px' },
+              { label: 'XL', value: '30px' },
+            ],
+          },
+          defaultValue: '24px',
+        },
+        {
+          type: 'text-editor',
+          label: 'Text',
+          name: 'descriptionText',
```

Comment:
I think `textarea` should be the input type for description, no need to use a richtext editor in this case